pkg/operator/ceph/cluster/cluster.go

```diff
@@ -209,7 +210,9 @@ func (c *cluster) createInstance(rookImage string, cephVersion cephver.CephVersi
 			return fmt.Errorf("failed to start the rbd mirrors. %+v", err)
 		}
 
-		logger.Infof("Done creating rook instance in namespace %s", c.Namespace)
+		msg := "Done creating rook instance in namespace %s"
```

Comment:
Could you change this message to the following? Since it's going to be visible in the status it will be more descriptive:
`Done creating or updating ceph cluster in namespace %s`